src/core/lv2/Lv2Proc.cpp

```diff
@@ -496,8 +521,17 @@ void Lv2Proc::initMOptions()
 
 void Lv2Proc::initPluginSpecificFeatures()
 {
+	// options
 	initMOptions();
 	m_features[LV2_OPTIONS__options] = const_cast<LV2_Options_Option*>(m_options.feature());
+
+	// worker (if plugin has worker extension)
+	Lv2Manager* mgr = Engine::getLv2Manager();
+	if (lilv_plugin_has_extension_data(m_plugin, mgr->uri(LV2_WORKER__interface).get())) {
+		m_features[LV2_WORKER__schedule] = m_worker->feature();
```

Comment:
At this point, `m_worker` is an empty optional, so calling `operator->` results in undefined behaviour.

Reply:
I tried to fix this by instantiating the worker already before this line, but without the not-available handle yet, and pass the handle later.